src/plugins/good-meetup-tracking.test.js

```diff
@@ -79,9 +77,7 @@ describe('Integration with tracking logs', () => {
 	});
 
 	it('encodes standard output from logTrack', () => {
-		const tracker = new GoodMeetupTracking({
-			postData() {},
```

Comment:
this is no longer needed since by default `postData` will no-op unless `NODE_ENV=production`. Leaving the default behavior in place allows us to force the test to hit the real endpoint with `NODE_ENV=production yarn test`